aws/config.go

```diff
@@ -45,10 +46,14 @@ func ConfigFromURL(awsURL *url.URL) (*aws.Config, error) {
 	}
 
 	if strings.Contains(awsURL.Host, ".") {
+		region := "dummy"
+		if os.Getenv("AWS_REGION") != "" {
+			region = os.Getenv("AWS_REGION")
```

Comment:
I raised an eyebrow at the two identical calls - what about:

		region := os.Getenv("AWS_REGION")
		if  region == "" {
			region = "dummy"
		}

Reply:
Perfect!